tests/integration/targets/rds_instance/inventory

```diff
@@ -14,6 +14,7 @@ modify
 tagging
 replica
 upgrade
+iam_roles
 
```

Comment:
You may also want to increase this to allow the extra tests to run in parallel:
https://github.com/ansible-collections/community.aws/blob/main/tests/integration/targets/rds_instance/main.yml#L9

The RDS tests are very very slow !